### ISSUE TITLE
Bitwise operators not supported?

### ISSUE THREAD

**Issue by oriolarcas** [basil00/Divert#265]
I was trying to implement filters for IP network masks, and it seems that bitwise AND or OR operators are not supported. It came as a surprise, since I think this should be relatively straightforward to implement (new "&" and "|" bitwise operators) and would be a valuable feature.

Is it already implemented and I got it wrong? Otherwise, is there a workaround, besides doing it in userspace?

Great project, congratulations anyway!

**Comment by oriolarcas** [comment]
In case somebody else is looking into this, finally I will be using the comparison operators "<=" and ">=" to check IP network ranges.

E.g., to check for 192.168.1.0/24, I will use the filter:

```
ip.DstAddr >= 192.168.1.0 and ip.DstAddr <= 192.168.1.255
```

**Comment by basil00** [comment]
No, bitwise or other operators are not supported, and I agree this would be a nice addition to the filter language.  I can keep this open as a feature request, but it would take some effort to implement, so might not added for a while.
